compiler/Makefile

```diff
@@ -176,6 +178,7 @@ generate_code:
 	@echo "Nexus Compiler: Moving files to output directory"
 	cp -r _generated/{client,apis,crds,common,nexus-client,helper,nexus-gql} ${GENERATED_OUTPUT_DIRECTORY}
 	cp -r ${GOPATH}/src/nexustempmodule/nexus-gql/graphql.so ${GENERATED_OUTPUT_DIRECTORY}/nexus-gql
+	chmod -R 0777 ${GENERATED_OUTPUT_DIRECTORY}
```

Comment:
This was added due to the permission issue encountered when testing in linux 

cc: @ramramu3433 @LokeshvarKr 

> time="2023-01-13T22:51:39+05:30" level=error msg="\t > error: unable to unlink old 'build/crds/.gitkeep': Permission denied\n"
> time="2023-01-13T22:51:39+05:30" level=error msg="\t > error: unable to unlink old 'build/crds/cloudserviceregistry_cloudaccount.yaml': Permission denied\n"
>

Reply:
Is this for a local build on linux or is it build in container ?
@kacpersaw added some changes a bit back to mount with proper permissions.

Reply:
@LokeshvarKr evaluated DM with this fix https://gitlab.eng.vmware.com/nsx-allspark_users/watchmen/datamodel/-/merge_requests/99/diffs

It is working as expected. Hence removing `chmod -R 0777 ${GENERATED_OUTPUT_DIRECTORY}` this line